src/components/ContentNode/BlockVideo.vue

```diff
@@ -42,7 +42,7 @@ export default {
   margin-left: auto;
   margin-right: auto;
 
-  @include space-out-between-siblings($stacked-margin-xlarge);
+  @include space-out-between-siblings(var(--stacked-margin-xlarge));
```

Comment:
I haven't tested this yet, but I'm not 100% sure this will work as written, because it's very tricky to combine Sass functions and CSS properties.

Since Sass works as a preprocessor, it substitutes the actual value for the variables at build time whereas the CSS property is evaluated at runtime—I don't know of a way to combine them for this particular usage, but we would likely need to find a workaround for this change.

Reply:
It seems to be working according to my testing. Can you test this as well?

Reply:
You're right. That's surprising to me, but I'm glad it's working as-is!

Maybe the issue I was worrying about is only a limitation with Sass functions as opposed to mixins where the variables are directly substituted in rules—not sure.

Either way, very glad this works!